netket/operator/_continuous_operator.py

```diff
@@ -92,6 +92,12 @@ def __mul__(self, other):
         if isinstance(self, ContinuousOperator) and isinstance(other, float):
             from netket.operator import SumOperator
 
-            return SumOperator(self, coefficients=other)
+            return SumOperator([self], coefficients=[other])
         else:
             return NotImplemented  # pragma: no cover
+
+    def __sub__(self, other):
+        if isinstance(self, ContinuousOperator) and isinstance(
+            other, ContinuousOperator
+        ):
+            return self + (-1.0) * other
```

Comment:
```suggestion
            return self + ( - other)
```

and implement
```python
    def __neg__(self):
        return -1.0 * self
```
as well?

Reply:
makes sense